vectorsearch/resource_vector_search_endpoint.go

```diff
@@ -45,8 +47,13 @@ func ResourceVectorSearchEndpoint() common.Resource {
 			if err != nil {
 				return err
 			}
-			endpoint, err := wait.GetWithTimeout(d.Timeout(schema.TimeoutCreate))
+			endpoint, err := wait.GetWithTimeout(d.Timeout(schema.TimeoutCreate) - deleteCallTimeout)
```

Comment:
Technically, we should also subtract the time it took to make the create call. Otherwise, if it is longer than `deleteCallTimeout`, we will wait past the parent deadline.